nixos/modules/services/security/kanidm.nix

```diff
@@ -224,6 +225,7 @@ in
         Group = "kanidm";
 
         AmbientCapabilities = [ "CAP_NET_BIND_SERVICE" ];
+        BindReadOnlyPaths = defaultServiceConfig.BindReadOnlyPaths ++ certPaths;
```

Comment:
Could we use `lib.mkMerge` for the attrset instead of `//`? That would make this construct obsolete.